CMakeLists.txt

```diff
@@ -778,38 +777,36 @@ endif()
 
 if(WIN32)
   set(SYSTEM_LIBS ${SYSTEM_LIBS} shlwapi.lib rpcrt4.lib)
-  set(LIBS ${ROCKSDB_STATIC_LIB} ${THIRDPARTY_LIBS} ${SYSTEM_LIBS})
+  set(ROCKSDB_LIB ${ROCKSDB_STATIC_LIB})
```

Comment:
this change is included by https://github.com/facebook/rocksdb/pull/5916/commits/11273253f0bd62f4eeb485acf6709cb65f89dd62 already